node/src/manager/commands/analyze.rs

```diff
@@ -0,0 +1,22 @@
+use std::sync::Arc;
+
+use graph::{
+    components::store::EntityType,
+    prelude::{anyhow::anyhow, DeploymentHash, Error},
+};
+use graph_store_postgres::SubgraphStore;
+
+pub async fn analyze(
+    store: Arc<SubgraphStore>,
+    hash: String,
+    entity_name: String,
+) -> Result<(), Error> {
+    println!("Running ANALYZE for {entity_name} entity");
```

Comment:
Dumb question, do we have a proper logger for `graphman`?

Reply:
Yes, logging is hooked up for `graphman` and turned on by setting `GRAPH_LOG=debug` etc. But it's very annoying on the console.

Printing is the right way to produce information the user should see.